### PR TITLE
Update quick start and developer's guide

### PR DESCRIPTION
**Pull request by ICantCodeForAnything**
Add link to developer's guide in quick start guide for setting up one node stock only environment.
Add how to deploy functions to stock only environment.

### CODE REVIEW

File: docs/quickstart_guide.md [hunk 1]
```diff
@@ -259,6 +261,8 @@ for benchmarking asynchronous (i.e., Knative Eventing) case and more details abo
     > **BEWARE:**
     >
     > Deployer **cannot be used for Knative eventing** (i.e., asynchronous) workflows. You need to deploy them manually instead.
+    >
+    > Deployer uses YAML files defined in configs/knative-workload that are **specific** to firecracker. Please refer to the [Developer's Guide](developers_guide.md) for deploying to a Stock Knative environment.
```
Comment (ustiugov) [inline docs/quickstart_guide.md:265]:
```suggestion
    > Deployer uses YAML files defined in configs/knative-workload that are **specific** to firecracker. Please refer to the [Developer's Guide](developers_guide.md) for deploying functions in the container or gVisor based environment.
```

File: docs/quickstart_guide.md [hunk 2]
```diff
@@ -2,6 +2,8 @@
 This guide describes how to set up an _N_-node vHive serverless cluster with Firecracker MicroVMs.
 See [here][github-toc] to learn where to find table of contents.
 
+To see how to setup a one node cluster with stock-only or gVisor, see [Developer's guide](developers_guide.md).
```
Comment (ustiugov) [inline docs/quickstart_guide.md:5]:
```suggestion
To see how to setup a single node cluster with stock-only or gVisor, see [Developer's guide](developers_guide.md).
```

File: docs/developers_guide.md [hunk 3]
```diff
@@ -19,6 +19,10 @@ watch kubectl get pods -A
 kn service apply
 ```
 
+To deploy a function, please create and use your own YAML files following this guide:
+
+https://github.com/knative/docs/tree/main/code-samples/serving/hello-world/helloworld-python#yaml
```
Comment (ustiugov) [inline docs/developers_guide.md:24]:
```suggestion
To deploy a function in the stock or gVisor execution environment, please create and use your own YAML files following this [example](https://github.com/knative/docs/tree/main/code-samples/serving/hello-world/helloworld-python#yaml)
```

**Review by ustiugov — CHANGES_REQUESTED** [review]
Thank you for the contribution! Please apply the changes, squash the commits into one (with `-s` non-capital s option for git) as per docs/contributing...md doc explains.

**Comment by ICantCodeForAnything** [comment]
I've applied the changes and squashed the commits :D

**Comment by ustiugov** [comment]
Please fix the linter error: https://github.com/vhive-serverless/vHive/actions/runs/3376563816/jobs/5620875205#step:5:14

you need to sign off the commit with a particular format.

**Comment by ICantCodeForAnything** [comment]
Ok I think I've signed it off now

**Pull request merged**